hack/build-acceptable-bundles.sh

```diff
@@ -3,10 +3,34 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Function to remove everything after @ in the image reference
+remove_digest() {
+  local input="$1"
+  echo "$input" | sed 's/@.*//'
+}
+
+# Function to remove the checksum and leading hyphen from the image reference
```

Comment:
Leading hyphen?

Reply:
It removes the digest and file sha. I made it clearer in this commit https://github.com/konflux-ci/build-definitions/pull/1167/commits/691da0685efcabafa4a267caf6aa522a2daf9942